contributor/outline.md

```diff
@@ -1,64 +1,138 @@
+    - Understand how the host team would like you to reach out for help and questions while you're working.
+  - Development
+    - If you get stuck, try to unblock yourself by reading the project documentation or looking at other parts of the code in the project.
+    - After trying the above if you're still stuck then reach out to the host team for help.
+    It doesn't do either side any good for you to spin forever on a problem that could be resolved in a few-minute conversation with someone more knowledgable about the project.
+    It's OK to ask for help.
+    You aren't expected to be as knowledgable and proficient as you might be on the full-time projects that your team owns.
+    - Match the existing coding style, indentation, etc, even if it's not what you would normally do.
+    A guest contribution for a feature is not the time to introduce a new way of structuring or formatting project code.
+  - Submitting the pull request
+    - Test out your contribution before opening the pull request.
+      - Ideally you can create a modified build of the shared project with your changes and try it out in your own project that consumes it.
+    - Ensure that your pull request includes any documentation updates that are relevant to your changes.
+    - Be sure that your pull request includes just the relevant changes for the issue you're completing.
+    Making pull requests too broad and large makes them more difficult to review and longer before they're accepted.
+  
+## Benefits of Contribution
+
+  - Personally
+    - Puts your development time into a place where others are also working.
+    It's not just up to you to maintain this area.
+    - Get help and support from the host team in implementing your changes.
+    - Your reputation and impact expands to outside your team.
+    - Learn from engineers on the host team and their coding and working styles.
+    This extra learning comes as part of your day job - no stretch or overtime assignment needed.
+    - Varied, fun, and personally fulfilling.
+  - Team
+    - You can get improvements you need in other projects on a timeline that you control.
+    - High influence into the development, progression, and ecosystem of projects that you depend on.
+    - Relieved of the long-term burden of maintenance of your contributions.
+    The host team maintains them as part of maintaining the overall project.
+    - Key knowledge spread across multiple people and teams.
+    Improves the [bus factor].
+    - Do the right thing for the company by putting your work into shared projects that others can use instead of your own one-offs.
+  - Company
+    - Improved cross-company collaboration.
+    - Better overall project quality.
+    "With many eyes, all bugs are shallow"
+    - Reduced duplication of projects.
+    - Better alignment of shared projects to the needs of their internal consumers.
```

Comment:
We mention the career development, learning and fun at work effects for the individual, should we mention the potential improved people retention effect for the company here too?